common/src/main/scala/com/gu/media/logging/KinesisLogging.scala

```diff
@@ -1,38 +0,0 @@
-package com.gu.media.logging
-
-import ch.qos.logback.classic.spi.ILoggingEvent
-import ch.qos.logback.classic.{Logger => LogbackLogger}
-import com.gu.logback.appender.kinesis.KinesisAppender
-import com.gu.media.Settings
-import com.gu.media.aws.AwsAccess
-import org.slf4j.{LoggerFactory, Logger => SLFLogger}
-
-trait KinesisLogging { this: Settings with AwsAccess =>
-  private val rootLogger = LoggerFactory.getLogger(SLFLogger.ROOT_LOGGER_NAME).asInstanceOf[LogbackLogger]
-
-  def startKinesisLogging(sessionId: String): Unit = {
-    rootLogger.info("bootstrapping kinesis appender if configured correctly")
-
-    for {
-      _stack <- stack
-      loggingCredentials <- credentials.logging
```

Comment:
I think we should be able to remove other code which sets up these logging creds. I.e.https://github.com/guardian/media-atom-maker/blob/e707e35627e955c7b5e5ff5e9bfc2573138eafd5/common/src/main/scala/com/gu/media/aws/AwsCredentials.scala#L55-L60

Reply:
Great spot, have removed that and related code. Hopefully got everything!